modules/cluster-logging-rn-5.4.2.adoc

```diff
@@ -0,0 +1,64 @@
+[id="cluster-logging-release-notes-5-4-2"]
+= Logging 5.4.2
+This release includes https://access.redhat.com/errata/RHBA-2022:4874[RHBA-2022:4874-OpenShift Logging Bug Fix Release 5.4.2]
+
+[id="openshift-logging-5-4-2-bug-fixes"]
+== Bug fixes
+* Before this update, editing the Collector configuration using `oc edit` was difficult because it had inconsistent use of white-space.  This change introduces logic to normalize and format the config prior to any updates by the Operator so that it is easy to edit using `oc edit`. (link:https://issues.redhat.com/browse/LOG-2319[LOG-2319])
```

Comment:
```suggestion
* Before this update, editing the Collector configuration using `oc edit` was difficult because it had inconsistent use of white-space. This change introduces logic to normalize and format the configuration prior to any updates by the Operator so that it is easy to edit using `oc edit`. (link:https://issues.redhat.com/browse/LOG-2319[LOG-2319])
```